src/redux/configureStore.js

```diff
@@ -1,4 +1,5 @@
 import { configureStore } from '@reduxjs/toolkit';
+// import thunk from 'redux-thunk';
```

Comment:
Great job @AhmedNazirMusah 🤝

- [ ] [Optional] You may always remove commented code from your project as this is regarded as a bad practice.